wazo_phoned/plugins/api/http.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import logging
+import yaml
+
+from flask import make_response
+from xivo.chain_map import ChainMap
+from xivo.rest_api_helpers import load_all_api_specs
+
+from wazo_phoned.auth_remote_addr import AuthResource
+
+logger = logging.getLogger(__name__)
+
+
+class SwaggerResource(AuthResource):
```

Comment:
```suggestion
class OpenAPIResource(AuthResource):
```
OR
```suggestion
class APIResource(AuthResource):
```
Swagger doesn't refer anymore to this specification

Reply:
Fixed. Thanks!